### DISCUSSION TITLE
How to hide the arrow for v-tooltip or v-popover?

### DISCUSSION THREAD

**Discussion by AJB99** [Akryum/v-tooltip#674] [Q&A]
Hi there, thanks in advance for the great lib. I've run into a situation where I need to hide the arrow, and I can't seem to figure out how:

I've tried:
```
<v-popover
    popperOptions="{
        modifiers: {
            arrow: {
                enabled: false
            }
        }
    }"
>
...
</v-popover>
```

But that doesn't seem to do the trick.

How would I go about hiding the arrow?

**Comment by tiepnguyen** [comment]
I just simply tweak the css to hide the arrow
```css
.tooltip-arrow {
  display: none;
}
```

Any better options?
